modules/howtos/examples/TransactionsExample.java

```diff
@@ -102,7 +102,7 @@ static void create() {
         try {
             transactions.run((ctx) -> {
                 // 'ctx' is an AttemptContext, which permits getting, inserting,
-                // removing and replacing documents, performing N1QL queries, and committing or
+                // removing and replacing documents, performing SQL++ queries, and committing or
```

Comment:
We haven't yet started changing references of `N1QL` to `SQL++`, so I will revert this for now I'm afraid.